zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java

```diff
@@ -21,7 +21,9 @@
 import org.apache.zeppelin.common.JsonSerializable;
 
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
```

Comment:
The unnecessary imports of `java.util.HashSet` and `java.util.Set` were added accidentally. I'll remove them by the next commit.